docs/_includes/components/utility-bar.html

```diff
@@ -8,10 +8,10 @@ <h2 id="utility-bar">Utility bar</h2>
     <li><a href="#">App link 1</a></li>
     <li><a href="#">App link 2</a></li>
     <li>
-      <a href="http://logout.link.goes.here" data-toggle="id7:account-popover" data-loginlink="http://login.link.goes.here" data-name="Your name">Sign out</a>
+      <a href="http://logout.link.goes.here" data-mw-functionality="true" data-toggle="id7:account-popover" data-loginlink="http://login.link.goes.here" data-name="Your name">Sign out</a>
```

Comment:
Think we can probably remove this from the documentation as we'll make it on-by-default when we release